### DISCUSSION TITLE
Camera setting doesn't take effect to qglviewer

### DISCUSSION THREAD

**Discussion by Junang-Wang** [sofa-framework/sofa#4463] [Write / Run a simulation] [answered]
Hi, 
I positioned the camera in my scene, but the `qglviewer` doesn't have any effect. However, if I change `qglviewer` to `qt`, the camera object works. Can anyone help me solve this problem?

`rootNode.addObject('InteractiveCamera',name='c')
  rootNode.c.position.value = [0,0,800]`

Thank you for your help in advance.

**Comment by fredroy** [comment, marked as answer]
I think qglviewer does not take into account any Camera object in the scene, unfortunately 😓

**Reply by Junang-Wang** [reply]
Hi, @fredroy 
Thanks for your reply. Then one has to use qt viewer for setting up camera.
